### PR TITLE
Add Pagination, Sorting, and Swagger

### PR DESCRIPTION
**Pull request by dima-semeniuk**

### CODE REVIEW

**Review by MinoDentori — APPROVED** [review]
Well done!

File: src/main/java/mate/academy/webapp/dto/book/CreateBookRequestDto.java [hunk 1]
```diff
@@ -7,13 +7,13 @@
 
 @Data
 public class CreateBookRequestDto {
-    @NotNull
+    @NotNull(message = "can't be null.")
```
Comment (kshuryhin) [inline src/main/java/mate/academy/webapp/dto/book/CreateBookRequestDto.java:10]: it looks like these changes are being done in the scope of another PR, aren't they?

**Review by kshuryhin — CHANGES_REQUESTED** [review]
Good job, just some clarification regarding DTO validation

File: src/main/java/mate/academy/webapp/dto/book/CreateBookRequestDto.java [hunk 2]
```diff
@@ -16,8 +16,8 @@ public class CreateBookRequestDto {
     private String author;
     @NotNull(message = NOT_NULL)
     private String isbn;
-    @NotNull(message = GREATER_THAN_ZERO)
-    @Positive
+    @NotNull(message = NOT_NULL)
+    @Positive(message = GREATER_THAN_ZERO)
```
Comment (kshuryhin) [inline src/main/java/mate/academy/webapp/dto/book/CreateBookRequestDto.java:20]: That's a bit confusing, you already have these changes in the main branched, haven't you?

**Review by kshuryhin — CHANGES_REQUESTED** [review]

**Review by kshuryhin — APPROVED** [review]
looks good

**Pull request merged**